README.md

```diff
@@ -1,8 +1,11 @@
 # Beaker
 
-Beaker is an acceptance testing harness for Puppet PE and other Puppet Projects.  It can also be used as a virtual machine provisioner - setting up machines, running any configuration on those machines and then exiting.
+Beaker is an acceptance testing harness for Puppet PE and other Puppet Projects.
```

Comment:
+1. Beaker is too closely tied to Puppet, hence the community thinks it is only good for testing Puppet.